### PR TITLE
Adds 10 new departmental jackets/outerwear

### PR DESCRIPTION
**Pull request by TemThrush**
## About the PR 
This PR adds 10 unique department jackets/outerwear each found in their respective clothing vendor (or equipment vendor where applicable) 

(Sprites highlighted in yellow were made by TemThrush while sprites highlighted in pink were made by Sunkiisu, with various modifications from both of us.)
image

In-game item sprites
image

In-game turnaround with and without bags equipped 
image

## Why's this needed? 
Fun department-appropriate clothing is always a must and allows for more fun customization. Plus there are simply not enough puffy sleeves on this station and I am positively starving out here man...

## Changelog 

```changelog
(u)Temthrush & Sunkiisu
(*)Adds 10 different outerwear which can be found in department clothing/equipment vendors.
```

### CODE REVIEW

File: code/obj/item/clothing/suits.dm [hunk 1]
```diff
@@ -1837,6 +1837,98 @@ TYPEINFO(/obj/item/clothing/suit/space/industrial/salvager)
...
 
+/obj/item/clothing/suit/wintercoat/short_nurse_dress
+	name = "short nurse dress"
+	desc = "A short, easy to move in nurse dress. Give it a twirl!"
+	icon = 'icons/obj/clothing/overcoats/item_suit.dmi'
+	wear_image_icon = 'icons/mob/clothing/overcoats/worn_suit.dmi'
+	icon_state = "short-nurse-dress"
+	item_state = "short-nurse-dress"
+	body_parts_covered = TORSO|ARMS
+	hides_from_examine = C_UNIFORM
```
Comment (Sovexe) [inline code/obj/item/clothing/suits.dm:1848]:
```suggestion
```
inconsistent with others?

File: code/obj/item/clothing/suits.dm [hunk 2]
```diff
@@ -1837,6 +1837,98 @@ TYPEINFO(/obj/item/clothing/suit/space/industrial/salvager)
...
+	body_parts_covered = TORSO|ARMS
+
+/obj/item/clothing/suit/wintercoat/botany_puffer
+	name = "botanist puffer"
+	desc = "A big comfy puffer jacket, perfect for gardening!"
+	icon = 'icons/obj/clothing/overcoats/item_suit.dmi'
+	wear_image_icon = 'icons/mob/clothing/overcoats/worn_suit.dmi'
+	icon_state = "botanist-puffer"
+	item_state = "botanist-puffer"
+	hides_from_examine = C_UNIFORM
```
Comment (Sovexe) [inline code/obj/item/clothing/suits.dm:1903]:
```suggestion
	body_parts_covered = TORSO|ARMS
```
same

**Review by flrsh — CHANGES_REQUESTED** [review]
I am a fan of department-specific clothing. However, I have trouble identifying a lot of these jackets with which department / role they're supposed to be for. The bottom two left ones and the top two right ones are especially indistinct; the blue and green one looks like a jacket version of the botanist jumpsuit and the two medical ones look a lot more catering-esque to me. I think they need more work. The top two left ones, I think, made the best use of departmental colors and aesthetics. 

In general I also think these are quite bulky, which sometimes looks weird, especially with side sprites and different head / body types. Could you provide a couple examples of these when worn facing east / west?

edit: Also, forgot to ask, are you using a set palette with the same colors from other departmental clothing items, or just similarish colors?

**Comment by TemThrush** [comment]
> 

I can definitely work more on some of the jackets to make them more distinguishable, I think changing the stripes colors on the dress and adding more red to the medical jacket may help. As well as removing the blue from the geneticist coat, these are just what come to mind for me though, and if you have any suggestions I'd love to hear them! 

The bulk is largely on purpose just because there's a lack of that sort of clothing available and we thought it'd be fun and unique! If it's too much or causes issues it can certainly be toned down. 
image

Here are the side views! I can also provide them with more unique models if you think that would better display how they look on different body types. (I also removed the bags for clarity but if you'd rather have an example including them I can do that!)

As for colors most of them have the same colors as pre-existing departmental clothing, I believe the only exceptions are the Janitor's Jacket and the Ranchers (which comes close but is off by a little bit) The gold detailing on the security and science puffers aren't taken from any pre-existing palette either iirc.

Either way, I'll get to work on modifying them! 👍

**Comment by TemThrush** [comment]
> I am a fan of department-specific clothing. However, I have trouble identifying a lot of these jackets with which department / role they're supposed to be for. The bottom two left ones and the top two right ones are especially indistinct; the blue and green one looks like a jacket version of the botanist jumpsuit and the two medical ones look a lot more catering-esque to me. I think they need more work. The top two left ones, I think, made the best use of departmental colors and aesthetics. I also appreciate the variety of shapes and designs so you aren't relying solely on color / shading to tell what's what.
> 
> Though in general I also think these are quite bulky, which sometimes looks weird, especially with side sprites and different head / body types. Could you provide a couple examples of these when worn facing east / west?
> 
> edit: Also, forgot to ask, are you using a set palette with the same colors from other departmental clothing items, or just similarish colors?

I can definitely work more on some of the jackets to make them more distinguishable, I think changing the stripes colors on the dress and adding more red to the medical jacket may help. As well as removing the blue from the geneticist coat, these are just what come to mind for me though, and if you have any suggestions I'd love to hear them! 

The bulk is largely on purpose just because there's a lack of that sort of clothing available and we thought it'd be fun and unique! If it's too much or causes issues it can certainly be toned down. 
image

Here are the side views! I can also provide them with more unique models if you think that would better display how they look on different body types. (I also removed the bags for clarity but if you'd rather have an example including them I can do that!)

As for colors most of them have the same colors as pre-existing departmental clothing, I believe the only exceptions are the Janitor's Jacket and the Ranchers (which comes close but is off by a little bit) The gold detailing on the security and science puffers aren't taken from any pre-existing palette either iirc.

Either way, I'll get to work on modifying them! 👍

**Review by flrsh — APPROVED** [review]
These look great! Thank you for incorporating those changes and your work in general! ✨

**Comment by Tarmunora** [comment]
Even setting aside the question of why a space station has winter coats in the first place, why should the station come stocked with a _second_ set of departmental winter coats, especially ones that give the appearance of being bought from the local Space Sears:tm: rather than being company uniform?

**Comment by Glamurio** [comment]
> Even setting aside the question of why a space station has winter coats in the first place, why should the station come stocked with a _second_ set of departmental winter coats, especially ones that give the appearance of being bought from the local Space Sears™️ rather than being company uniform?

Because they look cute!

**Comment by TemThrush** [comment]
Not sure where to take it from here so I'm closing it for now!

**Pull request closed without merging**